Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni

```diff
@@ -121,5 +121,12 @@
 #string STR_OPTION_ROM_ENABLE_VGA   #language en-US  "Enable OptionROM loading only on GPUs"
 
 #string STR_ENABLE_CAMERA_PROMPT    #language en-US "Enable Camera"
-#string STR_ENABLE_CAMERA_HELP    #language en-US "If this option is disabled, the power to the USB webcam is disconnected completely"
+#string STR_ENABLE_CAMERA_HELP      #language en-US "If this option is disabled, the power to the USB webcam is disconnected completely"
 
+#string STR_BATTERY_START_THRESHOLD_PROMPT    #language en-US "Battery charging start threshold"
+#string STR_BATTERY_START_THRESHOLD_HELP      #language en-US "The battery will start charging once the charge level drops below this value."
+
+#string STR_BATTERY_STOP_THRESHOLD_PROMPT    #language en-US "Battery charging stop threshold"
+#string STR_BATTERY_STOP_THRESHOLD_HELP      #language en-US "The battery will stop charging once the charge level reaches this value."

```

Comment:
This lacks consistency with:  https://github.com/Dasharo/docs/pull/654

The option names were different I the test procedure and moreover ambiguous